### PR TITLE
docs: adds screen.logToTestingPlaygroundURL method

### PR DESCRIPTION
**Pull request by tsuki42**

### CODE REVIEW

**Comment by tsuki42** [comment]
Can someone point me in the right direction? Why have all the checks failed?

**Comment by MatanBobi** [comment]
Hi @tsuki42! Thanks for taking the time to put this one together :)
From the [Deploy preview step details](https://app.netlify.com/sites/testing-library/deploys/5fd982859b53180007e04b02)
you can see that it failed on the `build` step so I'd first try to run the command locally and see if it's working.
Though it looks like you've added [another backtick](https://github.com/testing-library/testing-library-docs/pull/715/files#diff-35b5e9e093aa2abc8d86718078452934a67b4501d74a0927a356902ccc7312acR207) to the code blocks so it might be that.. Give it a try :) I'd also recommend testing it locally to see that it's working as you expect..

**Comment by tsuki42** [comment]
Those extra backticks were added automatically during commit phase by lint-staged. I didn't even see them. My bad. I'll make the changes and try again. Thanks.

**Comment by MatanBobi** [comment]
> Those extra backticks were added automatically during commit phase by lint-staged. I didn't even see them. My bad. I'll make the changes and try again. Thanks.

No worries, if that's being added by our prettier configs we should fix that.
I see that it's still failing, did you manage to run the build command locally?

**Comment by tsuki42** [comment]
Those backticks were added again. I passed the `--no-verify` to bypass them and force-pushed the changes. Still failing.

**Comment by MatanBobi** [comment]
It looks like the prettier fixes messed up the line breaks for `Tabs` and `TabItem`..
We need to look into it, I'll try to get to it today and figure out why this happens.
For now, I pushed a fix to your branch :)

**Comment by tsuki42** [comment]
Yeah. I had reverted the changes back to master and redid the docs and the checks had passed.

**Comment by tsuki42** [comment]
Cleaned the commit messages. It is now ready for the review. Thanks.

**Review by nickmccurdy — APPROVED** [review]

File: docs/dom-testing-library/api-queries.mdx [hunk 1]
```diff
@@ -109,6 +109,27 @@ screen.debug(screen.getByText('test'))
 screen.debug(screen.getAllByText('multi-test'))
 ```
 
+### `screen.logTestingPlaygroundURL`
+
+For debugging using [testing-playground](https://testing-playground.com), screen
+exposes this convenient method which logs an URL that can be opened in a
```
Comment (MatanBobi) [inline docs/dom-testing-library/api-queries.mdx:115]: I think that the correct grammar for this one is "a URL" :)

**Review by MatanBobi — CHANGES_REQUESTED** [review]

File: docs/dom-testing-library/api-queries.mdx (hunk 1 above)
Comment (tsuki42) [reply docs/dom-testing-library/api-queries.mdx]: Oops! Fixed it. 😅

**Review by MatanBobi — APPROVED** [review]

**Comment by MatanBobi** [comment]
Thanks for this one @tsuki42!

**Comment by MatanBobi** [comment]
@all-contributors please add @tsuki42 for docs

**Comment by tsuki42** [comment]
> Thanks for this one @tsuki42!

My pleasure.

**Pull request merged**